src/components/standalone/firewall/CreateOrEditPortForwardDrawer.vue

```diff
@@ -165,13 +165,25 @@ watchEffect(() => {
 })
 
 function close() {
+  if (
```

Comment:
Not strictly needed, but if the error is given by a failed request of the form's options then clearing it could be confusing if the user reopens it again (they would see no error notification, but the options still aren't available). Maybe at this point we should simply refetch the options every time the drawer is opened, in case of an error?

Reply:
Yes, if the drawer need some data we usually fetch it every time it opens, so we can get updated data. If the code doesn't currently work this way we can postpone these changes to another PR, it's not an essential feature at the moment.

Reply:
Ok, I've updated the drawer so that it refetches the data every time it is shown, now it should be ok. I've also removed the aforementioned condition because of this change.